### PR TITLE
Scripts/Spells: Ultra-Advanced Proto-Typical Shortening Blaster

### PR DESCRIPTION
**Pull request by Sorikoff**
**Changes proposed:**
Migrate Ultra-Advanced Proto-Typical Shortening Blaster to scripts system.

**Target branch(es):** 3.3.5

**Tests performed:** Works as it did.

### CODE REVIEW

**Comment by Aokromes** [comment]
ci/circleci: nopch — Your tests failed on CircleCI

**Comment by jackpoz** [comment]
Please fix the merge conflicts of https://github.com/TrinityCore/TrinityCore/pull/23463 first, after I merge that we can see about this one too

File: src/server/game/Spells/SpellEffects.cpp [hunk 1]
```diff
@@ -3744,33 +3744,6 @@ void Spell::EffectScriptEffect(SpellEffIndex effIndex)
 
                     return;
                 }
-                case 45668:                                 // Ultra-Advanced Proto-Typical Shortening Blaster
-                {
-                    if (!unitTarget || unitTarget->GetTypeId() != TYPEID_UNIT)
```
Comment (jackpoz) [inline src/server/game/Spells/SpellEffects.cpp]: unitTarget->GetTypeId() != TYPEID_UNIT means that the target can only be a Creature but the new code doesn't have this condition. Is that ok or should this condition be added to the new script too ?

Comment (Sorikoff) [reply src/server/game/Spells/SpellEffects.cpp]: Spell that triggers this spell already has conditions tied to specific entries:
```sql
mysql> SELECT * FROM conditions WHERE SourceEntry=51912;
+-------------------------+-------------+-------------+----------+-----------+--------------------------+-----------------+-----------------+-----------------+-----------------+-------------------+-----------+-------------+------------+----------------------------------------------------------------+
| SourceTypeOrReferenceId | SourceGroup | SourceEntry | SourceId | ElseGroup | ConditionTypeOrReference | ConditionTarget | ConditionValue1 | ConditionValue2 | ConditionValue3 | NegativeCondition | ErrorType | ErrorTextId | ScriptName | Comment                                                        |
+-------------------------+-------------+-------------+----------+-----------+--------------------------+-----------------+-----------------+-----------------+-----------------+-------------------+-----------+-------------+------------+----------------------------------------------------------------+
|                      17 |           0 |       51912 |        0 |         0 |                       31 |               1 |               3 |           25434 |               0 |                 0 |         0 |           0 |            | Spell Crafty's Ultra-Advanced Proto-Typical Shortening Blaster |
|                      17 |           0 |       51912 |        0 |         1 |                       31 |               1 |               3 |           25432 |               0 |                 0 |         0 |           0 |            | Spell Crafty's Ultra-Advanced Proto-Typical Shortening Blaster |
+-------------------------+-------------+-------------+----------+-----------+--------------------------+-----------------+-----------------+-----------------+-----------------+-------------------+-----------+-------------+------------+----------------------------------------------------------------+
```

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Pull request merged**